app/uk/gov/hmrc/eoricommoncomponent/frontend/domain/registration/UserLocation.scala

```diff
@@ -31,16 +31,15 @@ object UserLocation {
 
   val validLocations: Set[String] = Set(Uk, Iom, Eu, ThirdCountry, ThirdCountryIncEU, Islands)
 
+  private val rowLocations = Set(Eu, ThirdCountry, ThirdCountryIncEU, Islands)
+
   def forId(locationId: String): Boolean = validLocations(locationId)
 
-  def isRow(requestSessionData: RequestSessionData)(implicit request: Request[AnyContent]) =
+  def isRow(requestSessionData: RequestSessionData)(implicit request: Request[AnyContent]): Boolean =
     requestSessionData.selectedUserLocation match {
-      case Some(UserLocation.Eu) | Some(UserLocation.ThirdCountry) | Some(UserLocation.Islands) | Some(
-            UserLocation.ThirdCountryIncEU
-          ) =>
-        true
-      case _ => false
+      case Some(location) => isRow(location)
+      case _              => false
     }
 
-  def isRow(location: String) = location != Uk
+  def isRow(location: String): Boolean = rowLocations.contains(location)
```

Comment:
Hmm, what is the usage of this method?

I'm just wondering because previously it checked everything except Uk (so if somehow we will pass Iom this will return true.

Right now, Iom is not as rowLocations. I would recommend to make this method private to be sure that we're using only isRow with RequestSessionData ourside of this class to be sure that we didn't change the current application behavior.

Not sure if I described this correctly, but happy to have a quick call to discuss this if required.

Reply:
I commented on this changing in the earlier discussion on UserLocation - 
"The other isRow method was only used for Registration journey which would have excluded Iom anyway. Hence the refactoring there."
But happy to chat.

Reply:
Fair enough. I missed that comment.